front_end/html/view_pp.html

```diff
@@ -0,0 +1,27 @@
+{% if course_basics["exists"] %}
+    <h2>Pair Programming Assignments</h2>
+    <p>
+      The following pairs will be used for turning in assignments in this course. Pairings are randomized and reset each week.
```

Comment:
Please change the wording to: "The following pairs have been assigned for this week. These pairings were assigned at random and are reassigned each week."